tests/estimators/keras_sentiment_classifier/keras_sentiment_classifier.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+"""Utility functions for working with an IMDB sentiment classifier."""
+
+import os.path
+
+import numpy as np
+import keras
+from keras.datasets import imdb
+from keras.preprocessing.sequence import pad_sequences
+from keras.preprocessing.text import text_to_word_sequence
+
+
+base_path = os.path.dirname(os.path.abspath(__file__))
+MODEL = os.path.join(base_path, 'keras_sentiment_classifier.h5')
+
+
+vocablen = 10000
+maxlen = 128
+
+
+word_index = imdb.get_word_index()
+word_index = {k:v+3 for k, v in word_index.items()}
+word_index['<PAD>'] = 0
+word_index['<START>'] = 1
+word_index['<OOV>'] = 2
+word_index['<UNUSED>'] = 3 # need this to be consistent with the vectorized IMDB dataset
+word_index = {k:v for k, v in word_index.items() if v < vocablen}
+
+
+reversed_word_index = {v:k for k, v in word_index.items()}
+
+
+def prepare_train_test_dataset():
+    np_load_old = np.load
+    np.load = lambda *a,**k: np_load_old(*a, allow_pickle=True, **k)
+    (x_train, y_train), (x_test, y_test) = imdb.load_data(num_words=vocablen,
+                                                          seed=113,
+                                                          start_char=1,
+                                                          oov_char=2,
+                                                          index_from=3, # <-- this does not seem to work?
```

Comment:
Probably would be nice to clarify this comment - I see that above we do ``v + 3``, is ``index_from=3`` still required, or it's a workaround? Also does code below work ok despite this line not having an effect?

Reply:
Just tested loading the dataset, it seems that `index_from` is the index *after* which words begin, i.e. 3 is not a valid index for a word, but 4 is. The line has no effect since 3 is the default.

Clarified the comment.

Reply:
Great, thanks for checking 👍